frontend/src/interfaces/address.ts

```diff
@@ -0,0 +1,9 @@
+export interface IAddress {
```

Comment:
I generally lean towards naming the file the same as the primary export, but it isn't necessary.